### ISSUE TITLE
Add a card for the marketplace

### ISSUE THREAD

**Issue by cryptoguard** [particl/particl.io#44]
What do you guys think about adding a card for the marketplace on the front page? Seems like it would be an easy way to put additional focus on the marketplace, these cards are supposed to enumerate our biggest features yet miss the marketplace (even though there's a brief section about it below)
![selection_016](https://user-images.githubusercontent.com/27031337/37079816-ea1d578e-21b1-11e8-9dd9-7dfabc2401b0.png)

**Comment by AllienWorks** [comment]
If you're up for writing a card for it, I'll be happy to add it.

**Comment by cryptoguard** [comment]
What about this? Up for discussion:

---

Decentralized & Private Marketplace
Particl's decentralized marketplace is built with privacy at its core and offers a low-cost, secure and scalable e-commerce experience. Designed to work with almost any coin, it presents itself as a solution for many different cryptocurrency, local and online communities to buy, sell and trade goods and services in the currency of their choice in a way that natively respects their rights to privacy.

**Comment by AllienWorks** [comment]
I would rephrase the second sentence (sounds a bit repetitive) to:

> Particl Market is a solution for many local and online communities to buy, sell and trade goods and services in the cryptocurrency of their choice in a way that natively respects their rights to privacy.

I'd include a mention of the privacy tools available (encrypted chat, 3 levels of privacy TXs etc.). But then again, I'm no copywriter :)

**Comment by cryptoguard** [comment]
Ok here we go, alternative version: 

Particl's decentralized marketplace is built with privacy at its core and offers a low-cost, secure and scalable e-commerce experience. Designed to work with almost any coin, it uses several privacy-enhancing features such as default private transactions, encrypted data, secure messaging, IP address anonymization, metadata stripping, private listings and more. It is exclusively governed by the Particl community which also earns all the fees it generates.

**Comment by cryptoguard** [comment]
Or maybe the last sentence sounds better this way: 

Particl's decentralized marketplace is built with privacy at its core and offers a low-cost, secure and scalable e-commerce experience. Designed to work with almost any coin, it uses several privacy-enhancing features such as default private transactions, encrypted data, secure messaging, IP address anonymization, metadata stripping, private listings and more. It is exclusively governed by the Particl community which also earns **all the fees generate by the marketplace.**

**Comment by cryptoguard** [comment]
Any feedback? 

@AllienWorks

**Comment by AllienWorks** [comment]
Yes, I really like the last one:

> Particl's decentralized marketplace is built with privacy at its core and offers a low-cost, secure and scalable e-commerce experience. Designed to work with almost any coin, it uses several privacy-enhancing features such as default private transactions, encrypted data, secure messaging, IP address anonymization, metadata stripping, private listings and more. It is exclusively governed by the Particl community which also earns all the fees generated by the marketplace.

I'd love to get approval from more team members before going live with it.

**Comment by AllienWorks** [comment]
@gerlofvanek can we get an illustration for marketplace please (in the same style as other ones in the carousel)?

**Comment by AllienWorks** [comment]
Here's a preview:

![](https://i.imgur.com/lco6lBV.png)

Waiting for approval.
